src/components/community/CommunityPage.jsx

```diff
@@ -0,0 +1,151 @@
+        const commitsResponse = await axios.get(`https://api.github.com/repos/${owner}/${repo}/commits`, { headers });
+        const issuesResponse = await axios.get(`https://api.github.com/repos/${owner}/${repo}/issues?state=all`, { headers });
+        const contributorsResponse = await axios.get(`https://api.github.com/repos/${owner}/${repo}/contributors`, { headers });
+
+        setRepoData({
+          stars: repoResponse.data.stargazers_count,
+          forks: repoResponse.data.forks_count,
+          openIssues: repoResponse.data.open_issues_count,
+          closedIssues: issuesResponse.data.filter(issue => issue.state === 'closed').length,
+          openPRs: prsResponse.data.filter(pr => pr.state === 'open').length,
+          closedPRs: prsResponse.data.filter(pr => pr.state === 'closed').length,
+          commits: commitsResponse.data.length,
+          contributors: contributorsResponse.data.length,
+        });
+        const topContributorsResponse = await axios.get(`https://api.github.com/repos/${owner}/${repo}/contributors?per_page=5`, { headers });
+        setContributors(topContributorsResponse.data);
+        setLoading(false);
+      } catch (error) {
+        console.error('Error fetching repo data:', error);
+        setLoading(false); 
+      }
+    };
+    fetchData();
+  }, []);
+  useEffect(() => {
+    window.scrollTo(0, 0); 
+  }, []);
+
+  const viewAllContributors = () => {
+    window.location.href = `https://github.com/${owner}/${repo}/contributors`;
+  };
+
+  return (
+    <div className="community-page">
+      {loading ? (
+        <div className="loading-spinner">
+          <ClipLoader color="#318f5f" loading={loading} css={override} size={150} />
+        </div>
+      ) : (
+        <div className="community-container">
+          <h1 className="title">EmoWell's Community</h1>
+          <div className="grid">
+            <div className="card">
+              <h2>Stars</h2>
+              <p>{repoData.stars}</p>
+            </div>
+            <div className="card">
+              <h2>Forks</h2>
+              <p>{repoData.forks}</p>
+            </div>
+            <div className="card">
+              <h2>Commits</h2>
+              <p>{repoData.commits}</p>
+            </div>
+            <div className="card">
+              <h2>Open Issues</h2>
+              <p>{repoData.openIssues}</p>
+            </div>
+            <div className="card">
+              <h2>Closed Issues</h2>
+              <p>{repoData.closedIssues}</p>
+            </div>
+            <div className="card">
+              <h2>Open PRs</h2>
+              <p>{repoData.openPRs}</p>
+            </div>
+            <div className="card">
+              <h2>Closed PRs</h2>
+              <p>{repoData.closedPRs}</p>
+            </div>
+            <div className="card">
+              <h2>Contributors</h2>
+              <p>{repoData.contributors}</p>
+            </div>

```

Comment:
Could you also add a button after this section , saying "contribute in github" or something similar. Before the contributors section.

Reply:
Will do that